pkg/runtime/executor.go

```diff
@@ -131,3 +131,13 @@ func (e *Executor) Execute() (results Output, err error) {
 	// successful
 	return results, nil
 }
+
+// GetTotalPolicyCount will return the total count of policies in all engines
+func (e Executor) GetTotalPolicyCount() int {
+	policyCount := 0
+	for _, engine := range e.policyEngine {
+		opaEngine := engine.(*opa.Engine)
```

Comment:
The policy engine is not guaranteed to be OPA. Although it is the only engine we support now, it's not a safe assumption to make here... But since this function belongs in the policy package anyway, you should just be able to pull the total directly from the engine.

Reply:
This change was removed after adding summary to ViolationStore